docs/crr-to-public-cloud.md

```diff
@@ -0,0 +1,123 @@
+# CRR from CloudServer to public cloud - Workflow
+
+## Installation
+
+An endpoint ('awsbackend', for example) will be defined in the [location
+configuration](https://github.com/scality/S3/blob/master/locationConfig.json) of
+[CloudServer](https://github.com/scality/S3). Each location constraint in the
+configuration file has properties needed for performing operations for the
+location. For example, if replicating to AWS S3, the properties will include the
+bucket name, region, and credentials for AWS S3. This enables the multiple
+backend feature in CloudServer that Backbeat uses for CRR to a public cloud.
```

Comment:
You could add a link somewhere in there to the S3 doc, to point them to the full info :) : http://s3-server.readthedocs.io/en/latest/USING_PUBLIC_CLOUDS/